src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryServiceV1.java

```diff
@@ -54,154 +53,69 @@ public void setAppName(final String name)
   }
 
   @Override
-  public void reportKafkaStart(final long startTime, final int maxTasks)
-  {
-    ObjectNode msg = MAPPER.createObjectNode();
```

Comment:
Is this code not used any more ?

Reply:
This is still in use, it is in line 70 in the file. Git diff marked this function as a deletion, which is not accurate. It is the same case with reportKafkaStop.